hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/fs/ozone/TestHSync.java

```diff
@@ -173,6 +173,7 @@ public static void init() throws Exception {
     // Reduce KeyDeletingService interval
     CONF.setTimeDuration(OZONE_BLOCK_DELETING_SERVICE_INTERVAL, 100, TimeUnit.MILLISECONDS);
     CONF.setTimeDuration(OZONE_DIR_DELETING_SERVICE_INTERVAL, 100, TimeUnit.MILLISECONDS);
+    CONF.setBoolean("ozone.fs.hsync.enabled", true);
```

Comment:
```suggestion
    CONF.setBoolean(OZONE_FS_HSYNC_ENABLED, true);
```

Reply:
Seems to be already set:

https://github.com/apache/ozone/blob/328e14123889ebbfd06615ecf29dc75645fe1d2b/hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/fs/ozone/TestHSync.java#L171

Reply:
```suggestion
```

Reply:
Good catch. Thanks @jojochuang and @adoroszlai !